packages/nodes-base/nodes/MySql/v2/helpers/utils.ts

```diff
@@ -190,6 +190,14 @@ export function prepareOutput(
 
 	return returnData;
 }
+const END_OF_STATMENT = /;(?=(?:[^'\\]|'[^']*?'|\\[\s\S])*?$)/g;
```

Comment:
typo STATEMENT

Reply:
This fixed the bug, but I'm wondering if we should reconsider splitting SQL statements, especially using regexes.

- Without using a proper SQL parser, there might still be some edge cases we're missing
- It's not the most efficient to run a query per statement. From what I've seen, mysql2 does support executing multiple statements at once:
https://sidorares.github.io/node-mysql2/docs/examples/typescript/row-data/multi-statements (with this we would avoid having to parse/split the query)

Reply:
by default we run multiple statements at once, user has to explicitly change mode to independent or batch in order to change logic to per statement query
and we use statements for "Detailed Output", when we show query result for each statement